MvvmCross/Core/Binding/MvvmCross.Binding.csproj

```diff
@@ -32,6 +32,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="System.Collections.Concurrent">
+      <HintPath>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETPortable\v4.5\Profile\Profile7\System.Collections.Concurrent.dll</HintPath>
```

Comment:
Microsoft Concurrent nuget is not available for silverlight apps. And Mvvmcross.Binding it also targeted at silverlight. Should we update the targets to remove silverlight ?